src/content/guides/installing-optional-ingress-controller/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Installing an Ingress Controller"
+description = "How to install the nginx-ingress-controller using our Web UI"
+date = "2020-02-04"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+
+# Installing an Ingress Controller
+
+An ingress controller helps you expose your services to the outside world.
+
+Not all of our clusters come with an ingress controller pre-installed. We are
+currently in a shift where certain releases have a pre-installed ingress controller
+and certain releases require you to install one yourself.
+
+Eventually all releases will not ship with a built-in ingress controller. That
+allows you full control to choose what and how many ingress controllers you
+want to run on your cluster.
+
+### Which releases do not ship with an ingress controller?
+
+Currently only clusters on our AWS installations that have a release version
+newer than 10.0.0 ship without an ingress controller by default.
```

Comment:
```suggestion
newer than `10.0.0` ship without an ingress controller by default.
```

Reply:
I like it more highlighted

Reply:
Maybe consider rewording to:
Currently only clusters on our AWS installations ...ship _by default_without an ingress controller.